corehq/apps/app_manager/dbaccessors.py

```diff
@@ -103,30 +103,29 @@ def get_app(domain, app_id, wrap_cls=None, latest=False, target=None):
     return app
 
 
-def get_apps_in_domain(domain, full=False, include_remote=True):
-    """
-    Returns all apps(not builds) in a domain
+def get_full_non_remote_apps_in_domain(domain):
+    from .models import Application
+    docs = [row['doc'] for row in Application.get_db().view(
+        'app_manager/applications',
+        startkey=[domain, None],
+        endkey=[domain, None, {}],
+        include_docs=True
+    )]
+    apps = [get_correct_app_class(doc).wrap(doc) for doc in docs]
+    return [app for app in apps if not app.is_remote_app()]
 
-    full use applications when true, otherwise applications_brief
-    """
-    if full:
-        view_name = 'app_manager/applications'
-        startkey = [domain, None]
-        endkey = [domain, None, {}]
-    else:
-        view_name = 'app_manager/applications_brief'
-        startkey = [domain]
-        endkey = [domain, {}]
 
+def get_brief_apps_in_domain(domain, include_remote=True):
     from .models import Application
-    view_results = Application.get_db().view(view_name,
-        startkey=startkey,
-        endkey=endkey,
-        include_docs=True)
```

Comment:
It's pretty hard to tell just looking at the original code, but here is the root of the performance bug. (It's very minor in the grand scheme of things, but definitely wrong.) It was calling `applications/applications_brief` with `include_docs=true`, which entirely defeats the purpose of calling that view.